templates/tags.html

```diff
@@ -18,16 +18,16 @@ <h1 data-i18n="page.tags.title"></h1>
       <th:block th:if="${theme.config.tagAndCategory.tag_cloud}">
         <div class="tag-wordcloud-container">
           <script type="text/javascript" th:inline="javascript">
-            const tags = /*[[${tagFinder.listAll()}]]*/
+            (function() {
+              const tagData = /*[[${tagFinder.listAll()}]]*/ || [];
```

Comment:
可以直接在 thymeleaf 语法后面跟随 [] 即可，不需要使用 `||`。

```suggestion
              const tagData = /*[[${tagFinder.listAll()}]]*/ [];
```